lib/cached_network_image.dart

```diff
@@ -86,81 +90,177 @@ class CacheManager {
 
   CacheManager._();
 
-  SharedPreferences prefs;
-  Map<String, CacheObject> cacheData;
+  SharedPreferences _prefs;
+  Map<String, CacheObject> _cacheData;
+  DateTime lastCacheClean;
+
   static Object _lock = new Object();
 
   ///Shared preferences is used to keep track of the information about the files
   _init() async {
-    prefs = await SharedPreferences.getInstance();
+    _prefs = await SharedPreferences.getInstance();
 
     //get saved cache data from shared prefs
-    var jsonCacheString = prefs.getString("lib_cached_image_data");
-    cacheData = new Map();
+    var jsonCacheString = _prefs.getString("lib_cached_image_data");
```

Comment:
should CacheManage know that it caches images? `lib_cached_image_data` implies it knows.

Reply:
The key is lib_cached_image_data just because I called it that when I made this library. If we change it we should also add some migration and I didn't think that was worth it.